[2024-07-12T02:07:04.345+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-12T02:07:04.381+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parent_dag.trigger_raw_load_accounts manual__2024-07-11T20:36:59.640614+00:00 [queued]>
[2024-07-12T02:07:04.391+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parent_dag.trigger_raw_load_accounts manual__2024-07-11T20:36:59.640614+00:00 [queued]>
[2024-07-12T02:07:04.393+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-12T02:07:04.417+0530] {taskinstance.py:2330} INFO - Executing <Task(TriggerDagRunOperator): trigger_raw_load_accounts> on 2024-07-11 20:36:59.640614+00:00
[2024-07-12T02:07:04.422+0530] {standard_task_runner.py:63} INFO - Started process 57175 to run task
[2024-07-12T02:07:04.428+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'parent_dag', 'trigger_raw_load_accounts', 'manual__2024-07-11T20:36:59.640614+00:00', '--job-id', '1170', '--raw', '--subdir', 'DAGS_FOLDER/dag_parent.py', '--cfg-path', '/tmp/tmp0w_nww4p']
[2024-07-12T02:07:04.432+0530] {standard_task_runner.py:91} INFO - Job 1170: Subtask trigger_raw_load_accounts
[2024-07-12T02:07:04.485+0530] {task_command.py:426} INFO - Running <TaskInstance: parent_dag.trigger_raw_load_accounts manual__2024-07-11T20:36:59.640614+00:00 [running]> on host kali
[2024-07-12T02:07:04.584+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='parent_dag' AIRFLOW_CTX_TASK_ID='trigger_raw_load_accounts' AIRFLOW_CTX_EXECUTION_DATE='2024-07-11T20:36:59.640614+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-11T20:36:59.640614+00:00'
[2024-07-12T02:07:04.586+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-12T02:07:04.665+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:08:04.677+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:09:04.683+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:10:04.690+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:11:04.701+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:12:04.710+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:13:04.721+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:14:04.732+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-11 20:37:04.586490+00:00 to become allowed state ['success'] ...
[2024-07-12T02:14:36.383+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-12T02:14:36.414+0530] {process_utils.py:132} INFO - Sending 15 to group 57175. PIDs of all processes in the group: [57175]
[2024-07-12T02:14:36.416+0530] {process_utils.py:87} INFO - Sending the signal 15 to group 57175
[2024-07-12T02:14:36.416+0530] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-12T02:14:36.418+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-12T02:14:36.436+0530] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/operators/trigger_dagrun.py", line 228, in execute
    time.sleep(self.poke_interval)
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-07-12T02:14:36.462+0530] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=parent_dag, task_id=trigger_raw_load_accounts, run_id=manual__2024-07-11T20:36:59.640614+00:00, execution_date=20240711T203659, start_date=20240711T203704, end_date=20240711T204436
